erigon-lib/common/dbg/mem.go

```diff
@@ -0,0 +1,50 @@
+package dbg
+
+import (
+	"github.com/ledgerwatch/erigon-lib/metrics"
+	"github.com/shirou/gopsutil/v3/process"
+	"os"
+)
+
+var (
+	memRssGauge          = metrics.NewGauge(`mem_rss`)
+	memSizeGauge         = metrics.NewGauge(`mem_size`)
+	memPssGauge          = metrics.NewGauge(`mem_pss`)
+	memSharedCleanGauge  = metrics.NewGauge(`mem_shared_clean`)
```

Comment:
let's rename to `mem_shared{type="clean"}` then we can add/remove `type` filter in grafana to show/hide total.